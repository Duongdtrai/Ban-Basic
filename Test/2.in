45
22
48 56 39 47 51 61 20 45 63 46 28 72 64 48 50 54 56 54 24 36 74 57 
98
82 84 57 7 85 89 16 59 10 11 64 99 32 22 23 94 33 49 43 81 73 32 14 75 25 80 21 9 79 72 74 25 12 24 93 20 84 62 23 99 21 27 32 28 53 77 45 4 99 26 30 19 25 22 44 44 82 28 51 37 88 98 87 77 74 52 97 59 93 64 64 76 15 30 35 64 94 47 53 34 27 78 86 82 93 72 33 45 5 60 55 6 72 52 94 29 42 16 
95
22 80 92 77 74 37 50 5 54 65 55 45 57 10 47 2 72 23 11 78 25 51 9 89 67 99 22 90 88 11 69 3 75 3 94 85 77 25 53 73 3 44 67 3 3 18 29 86 35 7 95 67 17 68 71 35 40 40 12 34 33 25 53 71 48 86 22 25 57 49 30 26 50 91 42 3 58 90 97 85 80 57 26 74 56 79 56 34 39 49 87 53 12 17 83 
37
2 48 11 51 65 11 8 90 91 48 46 13 38 38 35 27 52 89 83 49 27 7 7 78 49 37 88 92 33 93 30 100 50 95 12 74 63 
34
14 73 9 12 54 94 21 64 3 40 92 64 44 12 94 70 63 13 47 66 39 72 59 26 31 80 34 50 63 29 14 94 10 93 
43
25 94 8 24 92 25 7 90 45 11 55 47 7 70 34 80 78 58 82 55 2 51 51 60 75 66 92 97 18 66 43 37 93 45 90 20 26 19 7 81 35 50 52 
79
30 10 17 66 58 100 61 85 68 30 70 22 78 30 38 23 83 69 6 69 2 81 22 79 59 47 61 80 88 86 95 32 4 43 69 65 86 19 90 75 80 93 98 93 20 69 47 56 94 50 78 95 23 14 9 75 54 30 36 71 59 20 4 49 91 81 56 13 62 34 7 25 43 82 62 11 13 72 43 
1
89 
40
42 41 53 100 66 46 2 18 66 92 89 54 11 25 33 95 22 26 23 38 85 50 98 90 46 50 15 91 53 75 37 12 78 12 85 47 52 68 15 98 
48
27 93 46 81 16 38 47 39 11 25 64 14 60 23 90 66 97 87 60 57 10 6 57 3 82 23 68 50 3 90 41 12 20 17 87 17 15 79 10 72 6 48 44 62 100 16 43 73 
96
19 74 77 86 7 52 37 83 91 15 42 87 90 66 42 97 95 50 75 100 91 81 58 64 71 45 18 32 53 11 78 84 89 41 2 67 23 83 3 57 98 37 8 36 37 13 23 6 24 87 17 63 20 35 99 60 28 7 69 67 92 20 11 44 60 95 18 72 9 67 50 26 53 100 13 61 18 20 84 96 39 5 40 67 54 48 96 56 24 98 33 18 61 47 74 53 
38
20 37 3 85 2 1 67 2 61 65 79 4 87 49 22 20 40 8 66 68 11 16 12 18 43 50 98 16 93 94 67 92 22 65 67 26 1 68 
36
75 50 90 26 86 27 32 85 83 10 35 48 20 57 84 11 52 85 74 69 81 8 4 1 15 89 55 76 73 81 67 70 88 3 15 34 
50
48 88 16 18 14 70 30 28 84 19 70 16 19 74 23 25 83 97 52 50 2 50 16 35 40 78 25 81 35 48 51 53 8 63 8 16 7 83 15 78 33 19 27 31 49 73 48 48 51 85 
25
63 88 40 55 87 80 4 61 79 6 90 14 14 54 13 82 16 29 74 74 53 62 92 29 44 
83
7 93 8 18 64 81 67 21 58 75 26 60 29 100 66 37 48 78 57 77 79 74 34 77 31 68 4 86 45 47 14 17 58 69 99 34 42 61 52 45 83 14 16 57 57 93 40 10 13 84 10 40 39 21 2 33 26 15 88 88 40 72 46 61 79 53 96 6 85 32 72 30 91 48 16 87 24 13 83 85 74 56 18 
32
79 44 83 7 30 21 82 65 53 87 40 8 99 2 76 66 83 88 46 56 83 71 56 59 82 66 89 59 91 82 16 79 
58
41 10 74 3 4 87 23 1 9 73 94 25 50 5 93 43 35 43 70 8 38 57 55 90 29 91 89 56 43 34 42 46 66 43 17 57 41 82 29 52 60 98 76 33 3 94 39 70 41 75 90 39 60 33 33 86 42 98 
19
63 4 60 42 75 87 87 61 89 89 16 88 51 19 45 40 56 2 20 
4
55 38 52 18 
94
19 85 76 9 13 52 35 13 81 85 43 32 72 69 87 39 89 99 29 74 32 61 97 72 82 92 71 40 49 4 25 2 39 31 26 3 54 8 61 41 48 96 61 78 64 37 68 77 50 99 94 43 58 54 85 28 58 38 49 53 63 7 62 63 77 4 21 27 51 46 66 19 22 54 3 47 69 28 19 23 44 25 7 66 61 43 61 72 4 79 38 95 35 59 
79
13 33 22 57 82 66 5 53 97 55 62 94 39 8 62 54 84 64 92 4 54 72 43 9 16 3 37 89 24 38 49 13 71 59 64 22 30 28 3 1 6 45 39 43 57 65 5 48 62 6 74 41 36 8 100 38 6 22 33 39 9 62 31 66 35 30 17 9 52 23 70 61 30 33 15 79 25 15 53 
68
66 44 40 25 9 88 80 31 34 96 54 54 88 55 55 14 29 35 56 41 43 34 81 44 85 57 61 16 34 99 98 5 17 30 21 56 40 49 66 1 45 82 2 1 35 100 57 38 19 27 16 19 82 70 84 4 30 74 73 26 36 39 69 68 32 11 5 65 
38
79 33 92 61 37 70 28 8 67 68 91 5 53 29 58 16 68 25 92 24 93 71 48 16 69 70 36 65 99 67 30 91 8 32 97 7 13 16 
98
6 45 11 21 94 27 26 50 23 71 50 19 21 73 54 53 27 47 82 19 56 12 8 11 54 92 45 100 69 5 61 77 100 35 31 84 26 46 35 58 52 46 80 47 57 50 73 63 38 49 99 51 75 53 77 32 34 76 13 74 13 1 87 96 87 28 99 97 91 70 69 9 66 86 80 62 57 73 83 62 44 4 23 65 86 29 85 14 87 15 87 25 17 91 99 94 97 55 
87
79 23 62 44 60 20 28 55 44 16 49 76 98 100 34 94 34 39 9 71 17 26 34 36 96 81 33 16 16 77 71 43 84 57 74 98 91 59 52 72 62 50 97 2 80 52 7 51 53 75 8 11 78 13 67 32 7 74 89 59 13 31 54 29 22 6 26 52 56 17 4 10 96 1 26 64 33 98 60 97 19 44 51 86 15 90 37 
51
44 23 30 27 59 15 66 16 20 50 51 30 26 28 45 30 46 89 14 37 45 41 26 20 79 42 4 48 16 86 53 22 5 25 39 73 91 43 85 44 26 45 67 20 88 18 97 49 51 81 20 
26
70 62 61 90 33 45 56 84 2 31 40 51 25 80 36 49 6 27 87 86 10 67 10 53 3 32 
42
31 44 93 89 52 70 2 98 62 10 89 64 57 96 42 37 89 68 97 25 91 82 68 29 55 25 90 43 11 21 6 67 81 72 64 90 35 44 21 6 41 17 
81
84 75 55 67 99 32 33 49 20 6 4 76 34 79 46 55 60 72 30 8 65 29 21 49 85 22 68 50 33 32 51 56 76 42 31 92 59 50 3 43 32 64 80 15 31 64 43 95 98 40 52 89 92 89 52 50 43 6 29 56 20 1 100 67 46 17 9 45 9 16 11 94 30 15 46 43 86 50 40 65 87 
33
74 44 58 26 84 29 38 11 89 25 51 14 63 37 73 9 54 40 13 17 37 70 100 73 16 97 16 10 8 19 34 48 28 
34
29 4 65 56 74 28 7 46 64 64 96 49 15 8 14 10 45 15 23 36 47 51 22 96 18 90 57 32 96 76 18 25 87 42 
37
79 69 33 53 92 45 10 33 73 17 7 14 92 90 99 90 34 22 76 85 49 96 7 22 42 71 30 59 57 51 34 31 6 56 83 1 33 
75
99 25 86 37 93 37 60 37 25 7 47 51 8 51 1 72 42 28 21 59 60 63 34 82 94 62 72 36 17 12 7 79 82 66 15 90 70 99 16 71 42 70 12 56 43 45 39 49 62 71 74 89 34 7 66 44 85 97 40 21 81 4 88 52 77 74 24 60 75 59 30 77 76 5 48 
82
37 58 10 91 53 10 42 49 48 4 17 98 48 80 6 69 21 27 72 97 75 73 67 69 23 43 63 65 54 14 52 25 71 55 44 9 74 64 71 5 3 77 70 47 18 53 83 37 78 56 77 11 21 54 71 4 30 87 94 95 5 93 85 33 97 88 100 81 81 27 17 14 45 75 40 65 45 28 51 7 60 87 
10
34 69 53 1 30 44 95 78 6 18 
34
49 41 22 43 91 21 38 95 31 16 21 43 52 78 45 69 99 33 23 48 20 15 22 6 68 28 68 20 46 58 16 78 76 34 
18
34 93 70 72 75 53 61 38 84 75 72 53 100 31 35 59 77 50 
26
73 20 8 96 92 21 94 41 86 95 89 23 91 37 55 6 100 23 90 67 41 2 67 71 50 54 
14
59 28 69 96 55 51 43 23 79 87 18 64 22 14 
10
28 83 75 98 25 43 14 18 48 67 
6
65 14 76 80 47 21 
100
42 67 93 20 18 74 94 21 81 91 80 54 49 32 74 20 2 50 19 54 86 90 9 99 39 5 18 65 47 32 65 98 81 77 3 64 95 69 93 47 38 41 55 69 6 76 70 48 35 87 83 96 30 90 51 98 18 84 42 29 82 80 85 1 99 45 41 76 33 70 82 96 32 44 71 52 72 88 29 70 27 22 62 24 69 18 81 8 93 41 72 44 49 58 61 21 39 56 80 36 
26
83 93 87 41 49 3 96 27 25 52 32 89 91 65 16 76 18 25 15 12 65 58 66 4 56 53 
15
53 79 19 1 97 1 54 28 8 68 38 60 86 46 66 
