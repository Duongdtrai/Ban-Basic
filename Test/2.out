17085136
7824273560
91967400
81545112
47807968
88688340
3176426968
0
215692276
217312866
12836816586
139900510
41586972
338455288
28571728
410775228
43204980
1006163316
7852152
2392
2201728140
3529567980
1267102560
71596680
9579126544
2643723348
444582870
36781956
454827444
6060969552
108978050
1753600
240251770
1369650456
3941200884
211000
76941510
5422080
39234468
4062492
948478
122060
18819573530
13273260
6781950
